### DISCUSSION TITLE
FEATURE webhook router

### DISCUSSION THREAD

**Discussion by 07pepa** [fastapi/fastapi#11767] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
no example code
```

### Description

basicly https://fastapi.tiangolo.com/advanced/openapi-webhooks/ but in router with limited methods 
i know i can do this with APIRouter but it will document 10 times better 

### Operating System

Other

### Operating System Details

os independend

### FastAPI Version

does not mater feature does not exist in fastapi

### Pydantic Version

does not matter

### Python Version

does not matter

**Comment by YuriiMotov** [comment]
Could you please elaborate on this?
I'll close it for now to keep the discussion clean, but feel free to re-open when you are ready to provide more info
